api/src/org/apache/cloudstack/api/response/AutoScaleVmProfileResponse.java

```diff
@@ -75,6 +75,7 @@
     @Parameter(name = ApiConstants.CS_URL,
                type = CommandType.STRING,
                description = "the API URL including port of the CloudStack Management Server example: http://server.cloud.com:8080/client/api?")
+    // leaving cloud.com reference above as it serves only as an example
```

Comment:
I agree that it is out of the scope. I just wanted to ask about the idea of starting using notations instead of real URLs as examples.

To tell you the truth I have already seen so many time people not thinking while seeing examples like this and using a copy/paste working mode that I try to avoid it sometimes, just to force people to change the parameters.

It is nothing that you need to change, I just so it, and I wanted to check your opinion ;)